### DISCUSSION TITLE
调用川虎助理模型会报错

### DISCUSSION THREAD

**Discussion by tjsemkk** [GaiZhenbiao/ChuanhuChatGPT#947] [General]
最新的版本部署完以后，调用川虎助理及川虎助理pro模型会报错。

module 'openai' has no attribute 'error'

怀疑是调用的openai模块问题。
环境：Python: 3.9.10  •  Gradio: 3.43.2    版本：20231110

**Comment by GaiZhenbiao** [comment]
请附上完整报错日志？

**Reply by tjsemkk** [reply]
更新langchain 到0.0.334版本就OK了。
原始requirements版本是：0.0.276
